README.md

```diff
@@ -47,6 +47,9 @@ NOTE: It may be helpful to create a directory that is separate from your normal
    - It may take some time for the server to be ready to accept requests
      (~30 seconds in some cases), and the site will likely respond with a 503
      error before then.
+     
+1. Register a user with the username `reddit`. In order for anonymous access
```

Comment:
This worked for me, but maybe we should specify that you're creating the user in the reddit UI at http://reddit.local/login?